manifests/machineconfigdaemon/clusterrole.yaml

```diff
@@ -31,4 +34,4 @@ rules:
   resources:
   - subjectaccessreviews
   verbs:
-  - create
+  - create
```

Comment:
nit: I think we prefer to have a newline at the end of files

Reply:
Will do!